src/main/java/net/sf/jabref/gui/importer/EntryFromPDFCreator.java

```diff
@@ -64,7 +64,7 @@ public boolean accept(File f) {
         PdfImporter pi = new PdfImporter(JabRefGUI.getMainFrame(), JabRefGUI.getMainFrame().getCurrentBasePanel(), JabRefGUI.getMainFrame().getCurrentBasePanel().getMainTable(), -1);
         ImportPdfFilesResult res = pi.importPdfFiles(Collections.singletonList(pdfFile.toString()));
         if (res.getEntries().size() == 1) {
-            return Optional.of(res.getEntries().get(0));
+            return Optional.ofNullable(res.getEntries().get(0));
```

Comment:
So `res.getEntries().get(0)` could be `null`? 
In this case I would prefer if `pi.importPdfFiles` is fixed. A null value in a list is very very counterintuitive.